Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -189,7 +189,7 @@ div.bocList div.bocListTableBlock div.bocListNavigator > span > input
   padding: var(--remotion-themed-spacing-small);
   margin: var(--remotion-themed-spacing-small) var(--remotion-themed-spacing);
   text-align: right;
-  width: calc(0.6em * var(--size));
+  width: calc(0.6em * var(--default-icon-size));
```

Comment:
Why did you change this code?
```suggestion
  width: calc(0.6em * var(--size));
```

Reply:
Because '--size' is not defined in any css file

Reply:
I know, It's only added during rendering. This is correct code. (we could always discuss a concept of "declaring" such injected variables to make Rider/Reshaprer shut up. But that's out of scope here.

Reply:
Oh, and if you check, `--size` is set to the digit count, it's not in PX.